src/lib/modules/glimpse-highlight.js

```diff
@@ -4,5 +4,6 @@ var hljs = require('../../../node_modules/highlight.js/lib/highlight.js');
 
 hljs.registerLanguage('cs', require('../../../node_modules/highlight.js/lib/languages/cs'));
 hljs.registerLanguage('sql', require('../../../node_modules/highlight.js/lib/languages/sql'));
+hljs.registerLanguage('javascript', require('../../../node_modules/highlight.js/lib/languages/javascript'));
```

Comment:
This actually doesn't appear to be needed; I see the same behavior without it.  It's here just for consistency.  `highlight.js` makes a distinction between JSON and JavaScript in terms of formatting; what we get from `console.log()` is technically not JSON but JavaScript (the only real difference is that the former quotes property names and the latter does not, but it doesn't seem to matter to the formatter anyway).